### PR TITLE
feat: Add more unit tests

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/registry/comparison/comparison.go [hunk 1]
```diff
@@ -83,17 +82,35 @@ func DownloadCSVFile(ctx context.Context, models Comparison) error {
...
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := os.Remove(fileName); err != nil {
+			log.Error("remove file is failed, err: %s", err.Error())
+		}
+	}()
+
+	file, err := os.Open(fileName)
```
Comment (FogDong) [inline pkg/registry/comparison/comparison.go:97]: Need to close the file.

File: pkg/registry/comparison/comparison.go [hunk 2]
```diff
@@ -126,18 +132,28 @@ func createCSVFile(ctx context.Context, metas []*ormbmodel.Model) error {
 	}
 	csvWrite.Flush()
 
-	if _, err := file.Seek(0, io.SeekStart); err != nil {
-		log.Errorf("Seek file is failed, err: %s", err.Error())
+	return nil
+}
+
+func responseDownloadFile(ctx context.Context, fileName string) error {
```
Comment (FogDong) [inline pkg/registry/comparison/comparison.go:138]:
```suggestion
func respondCSVFile(ctx context.Context, fileName string) error {
```

File: pkg/registry/comparison/comparison.go (hunk 1 above)
Comment (simon-cj) [reply pkg/registry/comparison/comparison.go]: Yeh, comments are addressed, PTAL

**Review by FogDong — COMMENTED** [review]
/lgtm

**Pull request merged**